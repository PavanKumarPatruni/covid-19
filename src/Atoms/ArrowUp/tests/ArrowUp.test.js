import React from 'react';
import { shallow, mount } from 'enzyme';

import ArrowUp from '../ArrowUp';

describe('ArrowUp component', () => {
  const props = {
    color: '',
    size: 24,
  };

  test('match snapshot correctly', () => {
    const wrapper = shallow(<ArrowUp {...props} />);
    expect(wrapper).toMatchSnapshot();
  });

  it('match snapshot correctly', () => {
    const wrapper = mount(<ArrowUp {...props} />);
    expect(wrapper).toMatchSnapshot();
  });
});
