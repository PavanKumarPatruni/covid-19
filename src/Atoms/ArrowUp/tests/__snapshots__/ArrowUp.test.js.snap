// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ArrowUp component match snapshot correctly 1`] = `
<svg
  fill=""
  height={24}
  stroke=""
  strokeLinecap="round"
  strokeLinejoin="round"
  strokeWidth="2"
  viewBox="0 0 24 24"
  width={24}
  xmlns="http://www.w3.org/2000/svg"
>
  <line
    x1="12"
    x2="12"
    y1="19"
    y2="5"
  />
  <polyline
    points="5 12 12 5 19 12"
  />
</svg>
`;

exports[`ArrowUp component match snapshot correctly 2`] = `
<ArrowUp
  color=""
  size={24}
>
  <svg
    fill=""
    height={24}
    stroke=""
    strokeLinecap="round"
    strokeLinejoin="round"
    strokeWidth="2"
    viewBox="0 0 24 24"
    width={24}
    xmlns="http://www.w3.org/2000/svg"
  >
    <line
      x1="12"
      x2="12"
      y1="19"
      y2="5"
    />
    <polyline
      points="5 12 12 5 19 12"
    />
  </svg>
</ArrowUp>
`;
